test/ciab

```diff
@@ -145,7 +143,7 @@ docker rm -f memsql-ciab
 
 docker run -d --init \
     --name memsql-ciab \
-    -e LICENSE_KEY=${LICENSE_KEY} \
+    -e LICENSE_KEY=${1} \
```

Comment:
if we don't pass the license in the test command, this part is not defined right?

Reply:
Yup